Marlin/ultralcd.h

```diff
@@ -69,7 +69,7 @@
   #define LCD_ALERTMESSAGEPGM(x) lcd_setalertstatuspgm(PSTR(x))
 
   #define LCD_UPDATE_INTERVAL 100
-  #define LCD_TIMEOUT_TO_STATUS 15000
+//  #define LCD_TIMEOUT_TO_STATUS 15000 // 3axis, moved to configuration.h
```

Comment:
Remove altogether. Don't leave commented-out stuff. Git precludes the need for this.